packages/react/src/components/Tab/Tab.stories.mdx

```diff
@@ -0,0 +1,49 @@
+import {ArgsTable, Source, Story, Canvas, Meta} from '@storybook/addon-docs';
+import dedent from 'ts-dedent';
+import StoryConfig from '../../../.storybook/story-config.ts';
+import {withDesign} from '../../../.storybook/utils.ts';
```

Comment:
Let's remove this import as it is unused in this story.

Reply:
Addressed with e6b871a1e3fcc10757c7fee869f972f0805c2847